docs/source/pre_application/testnet.rst

```diff
@@ -13,3 +13,25 @@ You can view it on `etherscan <https://rinkeby.etherscan.io/address/0xc3871e2c11
 .. attention::
 
     This testnet is currently being migrated to Threshold Network.
+
+
+Stake and Bond
+--------------
+You need some rinkeby testnet ETH and 40,000 T in a metamask wallet.
```

Comment:
```suggestion
You need some Rinkeby testnet ETH and 40,000 T in a metamask wallet.
```

Reply:
Also, I believe the minimum stake on Ibex is 15,000 T instead of 40,000 T. Should we outline that, so that there aren't "wasted" tokens? Or do we think that will be confusing?